scripts/main.js

```diff
@@ -71,4 +71,57 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+console.log("De afdeling sales heeft " + departments.sales.numberOfEmployees + " medewerkers")
+console.log(`De afdeling sales heeft " ${departments.sales.numberOfEmployees}  " medewerkers`)
```

Comment:
Volgens mij heb je, als je backticks gebruikt, geen aanhalingstekens meer nodig. Nu worden de aanhalingstekens ook in de console gelogd. Ik weet niet of dat je bedoeling was?